### DISCUSSION TITLE
Spotify API Update

### DISCUSSION THREAD

**Discussion by Coffeewelove** [gilbarbara/react-spotify-web-playback#179] [Q&A] [answered]
Hello everyone, 

about a week ago spotify released an update and since then "preview_url" etc is no longer usable for apps that are in "development mode". Apparently this player is also affected, because I couldn't play anything in the demo player. In the private project it was also not possible to play anything, it loads and loads like on the demo page. If it works for someone else, I am happy about any help.

**Comment by gilbarbara** [comment]
Hey @Coffeewelove 

I just tested the [demo](https://react-spotify-web-playback.gilbarbara.dev/), and it played normally. 
This library doesn't use the `preview_url` at all.
Do you have a premium account?

**Reply by Coffeewelove** [reply, marked as answer]
Okay, I got it to work in the project. Unfortunately not in the demo, I used Safari and Chrome. Premium is available. Thanks for the quick feedback though.

**Reply by gilbarbara** [reply]
I'll take a look at the demo, thanks.
